### PR TITLE
show review comment

### PR DESCRIPTION
**Pull request by haytranngoc**

### CODE REVIEW

**Comment by haytranngoc** [comment]
ready

File: resources/views/user/books/show.blade.php [hunk 1]
```diff
@@ -35,38 +41,27 @@
...
+                    @foreach($review->commentReviews as $comment)
                     <div class="media mt-4">
                         <img class="d-flex mr-3 rounded-circle" src="http://placehold.it/50x50" alt="">
                         <div class="media-body">
-                            <h5 class="mt-0"></h5>
+                            <h5 class="mt-0">{{ $comment->user->name }}</h5>
+                            <p>{{ $comment->content }}</p>
                         </div>
                     </div>
+                    @endforeach
```
Comment (trangkarol) [inline resources/views/user/books/show.blade.php]: Indent: 4 space

File: resources/views/user/books/show.blade.php [hunk 2]
```diff
@@ -35,38 +41,27 @@
...
-                <img class="d-flex mr-3 rounded-circle" src="http://placehold.it/50x50" alt="">
-                <div class="media-body">
-                    <h5 class="mt-0"></h5>
-                </div>
-            </div>
-
           <!-- Comment with nested comments -->
+            @foreach($book->bookReviews as $review)
             <div class="media mb-4">
                 <img class="d-flex mr-3 rounded-circle" src="http://placehold.it/50x50" alt="">
```
Comment (trangkarol) [inline resources/views/user/books/show.blade.php]: Indent: 4 space

**Comment by trangkarol** [comment]
commented

**Comment by haytranngoc** [comment]
ready

**Comment by haytranngoc** [comment]
ready

**Comment by trangkarol** [comment]
merged

**Pull request merged**